src/vmm/src/linux/vstate.rs

```diff
@@ -469,13 +439,10 @@ pub struct Vm {
     irqchip_handle: Option<Box<dyn GICDevice>>,
 
     #[cfg(feature = "amd-sev")]
-    sev: Option<AmdSev>,
+    tee: Option<AmdSnp>,
```

Comment:
Not blocking, but I'm wondering if this should be named `snp: ...`? It's going to get re-named once TDX and CCA land anyways

Reply:
Yea, will rename.